content/momentum/3/3-api/hooks-core-ec-ssl-ssl-ctx-fixup.md

```diff
@@ -55,7 +48,7 @@ A pointer to the closure.
 
 <dd>
 
-An `ec_ssl_ctx` struct. For documentation of this data structure see [“ec_ssl_ctx”](/momentum/3/3-api/structs-ec-ssl-ctx).
+This struct would be passed as-is to OpenSSL functions as the SSL_CTX parameter.
```

Comment:
If you look at the preview, the `SSL_CTX` parameter shows up as `sslctx` (without underline).
My guess of the table is to list the parameters. May it be a table or list, as long as the whole page read right.